lib/palindrome_check.rb

```diff
@@ -1,5 +1,26 @@
+require 'pry'
 # A method to check if the input string is a palindrome.
 # Return true if the string is a palindrome. Return false otherwise.
 def palindrome_check(my_phrase)
```

Comment:
Although your algorithm works, it alters the input parameter string and requires a lot of space to duplicate the string.
Can you think of an approach to check if a string is a palindrome or not by just examining it i.e. comparing elements but not altering the string or needing to create a duplicate of the string?